app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd_ls = command.split(" ")
+    if cmd_ls[1] == cmd_ls[2]:
```

Comment:
you check if command has 3 arguments before it, and raise error with message if not

Reply:
also check if command is correct